k8s/configuration/secret.yaml

```diff
@@ -94,8 +94,9 @@ type: Opaque
 data:
   # Base64-encoded HTTPS private key
   cert.key: "<b64 encoded HTTPS private key>"
-  # Base64-encoded HTTPS Signed Certificate or Certificate Chain
-  cert.pem: "<b64 encoded HTTPS Signed Certificate or Certificate Chain>"
+  # Base64-encoded HTTPS certificate chain
+  # including all intermediate certificates and the actual certificate
+  cert.pem: "<b64 encoded HTTPS certificate chain"
```

Comment:
It depends on the certificate provider whether you need the complete chain or not.
One of our certificate providers is not the root and hence we need a chain.
More info here: https://support.dnsimple.com/articles/what-is-ssl-certificate-chain/